protected/views/patient/view.php

```diff
@@ -56,7 +56,7 @@
 			<div class="row">
 				<div class="large-12 column">
 					<div id="no-practice-address" class="alert-box alert with-icon">
-						Patient has no GP practice address, please correct in PAS before printing GP letter.
+						Patient has no Practitioner practice address, please correct in PAS before printing Practitioner letter.
```

Comment:
Do we care about "correct in PAS"? I've raised a JIRA for it, and done a quick look, there seems to be about 30 places that it is referenced.